sdk/eventhub/azure-eventhubs/azure/eventhub/producer.py

```diff
@@ -298,16 +302,34 @@ def _error(outcome, condition):
         if outcome != constants.MessageSendResult.Ok:
             raise condition
 
+    def create_batch(self, max_size=None, partition_key=None):
+        """
+        Create an EventDataBatch object with max message size being max_message_size.
+        The max_message_size should be no greater than the max allowed message size defined by the service side.
+        :param max_message_size:
+        :param partition_key:
```

Comment:
Could we add types and descriptions of the parameters.

Reply:
Types and descriptions added.